<launch>

    <!-- Mobile robotarm node -->
    <node name="robot_arm" pkg="aidu_robotarm" type="robot_arm" output="screen" respawn="false">
        <!-- launch-prefix="gdb -ex run - -args" -->
        <param name="motor_port" value="motor_comm"/>
        <param name="motor_config" value="$(find aidu_robotarm)/config/mobile_robot_arm.xml"/>
        
    </node>

</launch>
